workshop/content/intermediate/templates/conditions/_index.md

```diff
@@ -0,0 +1,264 @@
+![condition-test-property](conditions/condition-test-property.png)
+
+You should see a view, as in the example shown next, showing that your instance type is `t2.micro`:
+![ec2-instance](conditions/ec2-instance.png)
+
+Similarly, you can run the same solution by passing the value of `EnvType` parameter as `prod`, and you will observe the instance type for provisioned EC2 instance will be `t2.small`.
+
+Congratulations! You have learned how to conditionally specify resource property types!
+
+### **Challenge**
+
+So far, you’ve learned how to use conditions with resources in your CloudFormation templates . Now let’s say you want to conditionally add an output in the [Outputs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/outputs-section-structure.html) section of your `condition-resource.yaml` CloudFormation template.
+
+**Task:** Describe an [Outputs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/outputs-section-structure.html) section in your template. Specify a Logical ID as `VolumeID`, and use the `Ref` [intrinsic function](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-instance.html#aws-properties-ec2-instance-return-values) to return the ID of your Volume. Now, update your existing CloudFormation Stack `cfn-workshop-condition-prod` with modified template to create an output based on `IsProduction` condition evaluation.
+
+{{%expand "Need a hint?" %}}
+
+* See the documentation for [Stack Output](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/outputs-section-structure.html#outputs-section-structure-examples), and define a `VolumeId` output in your template.
+* How do you conditionally create an output?
+{{% /expand %}}
+
+{{%expand "Want to see the solution?" %}}
+
+Append the following content to the `condition-resource.yaml` file:
+
+```yaml
+Outputs:
+   VolumeID:
+      Value: !Ref Volume
+      Condition: IsProduction
+```
+
+Next lets update your stack  `cfn-workshop-condition-prod`, navigate to the AWS CloudFormation [console](https://console.aws.amazon.com/cloudformation), and choose to  `cfn-workshop-condition-prod` stack:
+
+* In the CloudFormation console, select **Update stack**.
+* In **Prepare template**, select Replace current template **option**.
+* In **Template source**, select **Upload a template file**.
+* Choose the `condition-resource.yaml` template.
+* You can leave the value of `EnvType` parameter as `prod`. Choose **Next**.
+* You can leave other Configure stack options default, Choose **Next**.
+* Choose **Update stack**. You can view the progress of the stack being created in the CloudFormation console.
+* Wait until the stack creation is complete. Refresh the view in the console until you see your stack to be in the `UPDATE_COMPLETE` status.
+
+
+Browse to the `Outputs` section of the stack, and validate `VolumeID` appears as an output:
+![condition-prod-update](conditions/condition-prod-update.png)
```

Comment:
Please replace with a new image to reflect `ImageId` instead of `ImageID`.

Reply:
In the image, Outputs value shows VolumeID not ImageID.